ccf_transaction_processor/scripts/start_cchost.sh

```diff
@@ -32,5 +33,5 @@ EFILE="${F_SERVICEHOME}/logs/error.log"
 OFILE="${F_SERVICEHOME}/logs/output.log"
 
 cd ${F_SERVICEHOME}/run
-${CCHOST} --config ${F_SERVICEHOME}/etc/cchost.toml > $OFILE 2> $EFILE &
+${CCHOST} --config ${F_SERVICEHOME}/etc/cchost${1}.toml > $OFILE 2> $EFILE &
```

Comment:
Would be nice to add a comment somewhere what this ${1} is for; comes here a bit out of left field ...